plugins/DB.cpp

```diff
@@ -3,6 +3,38 @@
 #undef SLOGPREFIX
 #define SLOGPREFIX "{" << _args["-nodeName"] << ":" << getName() << "} "
 
+bool BedrockPlugin_DB::rewriteHandler(int actionCode, const char* table, string& newQuery) {
+// Hack to make logging work in static function
+#undef SLOGPREFIX
+#define SLOGPREFIX "{" << "unknown" << ":" << "DB" << "} "
+    SINFO("bedrock Running DB rewrite handler.");
+    switch (actionCode) {
+        case SQLITE_INSERT:
+            if (table == "chats"s) {
```

Comment:
this an expensify specific table, it should go in the expensify plugin that uses it.